### ISSUE TITLE
How do I reduce the vendors~main.js size so that it is now 40M+ for build

### ISSUE THREAD

**Issue by hejiao76** [jupyterlab/jupyterlab#7104]
How do I reduce the vendors~main.js size so that it is now 40M+ for build

**Comment by madsi1m** [comment]
We have the same issue, I think this file is big due to installed extensions and the fact it is one big file, which for us means lab takes a while to start on a slow internet connection. i think this can be increased if the vendors-main was smaller, or split into many little files so the browser can download them quicker in parallel.

Can the file be minified and/or split into many files?

**Comment by jasongrout** [comment]
The file is minified by default in 1.1.1 in production builds. There are other issues here about splitting it up more when you have lots of extensions.

Can you post the information asked for in the new issue template, such as jlab version, extensions installed, etc.?

**Comment by jasongrout** [comment]
FYI, it seems that jlab 1.1.1 ships with a ~9M vender js file (which appears to be minified to 4M if you do `jupyter lab build`).

**Comment by madsi1m** [comment]
noticed that today as well. With all my extentions the file is around 12mb and is minified. Which is much better than my 70+mb file.

**Comment by jasongrout** [comment]
Closing as resolved. Thanks!
